solidity/test/isms/OPStackIsm.t.sol

```diff
@@ -232,15 +225,12 @@ contract OPStackIsmTest is Test {
             (messageId)
         );
 
-        (uint240 nonce, uint16 verison) = Encoding.decodeVersionedNonce(
+        (uint240 nonce, uint16 verison) = decodeVersionedNonce(
             l2Messenger.messageNonce()
         );
-        uint256 versionedNonce = Encoding.encodeVersionedNonce(
-            nonce + 1,
-            verison
-        );
+        uint256 versionedNonce = encodeVersionedNonce(nonce + 1, verison);
 
-        bytes32 versionedHash = Hashing.hashCrossDomainMessageV1(
+        bytes32 versionedHash = hashCrossDomainMessageV1(
```

Comment:
imo this is a nice opportunity to do the prank and remove the forking but not gonna force it